src/admin-views/troubleshooting/common-issues.php

```diff
@@ -31,10 +30,10 @@
 				<div class="tribe-events-admin-faq__answer">
 					<?php
 						$label = '<a href="' . esc_url( $common_issue['link'] ) . '" target="_blank" rel="noopener noreferrer">' . esc_html( $common_issue['link_label'] ) . '</a>';
-						echo sprintf( $common_issue['solution'], $label );
+						printf( esc_html( $common_issue['solution'] ), wp_kses_post( $label ) );
```

Comment:
There is no need to `wp_kses_post` here, its too expensive to keep running this all the time.